src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/UseExceptionThrowHelpersTests.cs

```diff
@@ -0,0 +1,1078 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the MIT license.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Xunit;
+using VerifyCS = Test.Utilities.CSharpCodeFixVerifier<
+    Microsoft.NetCore.Analyzers.Runtime.UseExceptionThrowHelpers,
+    Microsoft.NetCore.Analyzers.Runtime.UseExceptionThrowHelpersFixer>;
+using VerifyVB = Test.Utilities.VisualBasicCodeFixVerifier<
+    Microsoft.NetCore.Analyzers.Runtime.UseExceptionThrowHelpers,
+    Microsoft.NetCore.Analyzers.Runtime.UseExceptionThrowHelpersFixer>;
+
+namespace Microsoft.NetCore.Analyzers.Runtime.UnitTests
+{
+    public class UseExceptionThrowHelpersTests
```

Comment:
Can we have a test where `using System;` is missing? I guess we should have AddImportsAnnotation on the result of `TypeExpressionForStaticMemberAccess` call.

Reply:
Good catch.  Will add.

Reply:
Actually, I'm not sure that's desired.  The fixer is only going to kick in if someone had ArgumentNullException in their test.  If they didn't import System, they would have had to have spelled out System.ArgumentNullException, and then the fixer will using the same when outputting ThrowIfNull.  That seems better than changing the style.  I can add a test, though.

Reply:
Makes sense